src/qml/views/timeline/Track.qml

```diff
@@ -186,6 +188,8 @@ Rectangle {
                     } else {
                         clip.originalX -= originalDelta
                     }
+                } else
```

Comment:
Missing a '{' at the end of this line

Reply:
Yeah, updated with Update Track.qml below.